### ISSUE TITLE
Catching errors before ‘init’ action

### ISSUE THREAD

**Issue by ArturMoczulski** [rollbar/rollbar-php-wordpress#73]
From Wordpress plugin directory forums:

```
Hi there!

I had a case of a fatal error not being reported to Rollbar at all. Looking at the plugin source, it seems like Rollbar PHP logging is initialized only at the init action:

\add_action(‘init’, array(&$this, ‘initPhpLogging’));

This is a pretty big problem, because a lot of stuff tends to happen before ‘init’. Wouldn’t it be possible to run the whole thing on ‘plugins_loaded’ instead?

In addition to that, it would be great if this plugin could be used as a mu-plugin. This would make it possible to catch errors from other (regular) plugins. (Sentry’s WP plugin has that feature, for example)

Thanks!
```

https://wordpress.org/support/topic/catching-errors-before-init-action/

**Comment by ArturMoczulski** [comment]
This has been implemented in `master` and will be included in the upcoming release.

To enable the plugin as a Must-Use plugin, copy `mu-plugin/rollbar-mu-plugin.php` file from the repo to your Muse-Use plugins directory (usually `wp-content/mu-plugins`).

https://docs.rollbar.com/docs/wordpress will need to include the above instructions on how to use `mu-plugin/rollbar-mu-plugin.php` when this gets released

**Comment by ArturMoczulski** [comment]
This got released in `v2.5.0`: https://github.com/rollbar/rollbar-php-wordpress/releases/tag/v2.5.0

Also, I added UI for switching between `Must-Use` plugin and not
